docker/lr-pb/shard_bam.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import gzip
+from collections import OrderedDict
+from math import ceil
+import pysam
+from construct import *
+
+from multiprocessing.pool import ThreadPool
+from functools import partial
+
+
+def write_shard(bam, sharding_offsets, zmw_counts_exp, prefix, index):
+    """
+    Write subsection of PacBio bam to a shard.
```

Comment:
I'd prefer just a little more specific, i.e. what a subsection means (could be random of left unspecified).

Reply:
Added.